maven-plugin-tools-annotations/src/main/java/org/apache/maven/tools/plugin/extractor/annotations/JavaAnnotationsMojoDescriptorExtractor.java

```diff
@@ -102,6 +106,36 @@
 
     private static final GroupKey GROUP_KEY = new GroupKey( GroupKey.JAVA_GROUP, 100 );
 
+    /**
+     * 
+     * @see <a href="https://docs.oracle.com/javase/specs/jvms/se19/html/jvms-4.html#jvms-4.1">JVMS 4.1</a>
+     */
+    private static final Map<Integer, String> CLASS_VERSION_TO_JAVA_STRING;
```

Comment:
We should add reference to this code from pom to remember that we need update together with asm

Reply:
Done in https://github.com/apache/maven-plugin-tools/pull/171/commits/4c8e2499919b75c2e1cd5a12e474dc65070f7d4f.